### PR TITLE
grafana screenshots

### PR DESCRIPTION
**Pull request by sanjeevghimire**
* Added grafana steps and screenshots

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,15 @@ The [sample application](https://github.com/IBM/sample.microservices.web-app) us
 
 ![Flow](images/architecture.png)
 
+## Flow
+
+1. Create Kubernetes service from IBM cloud.
+1. Deploy all microservices and web application also as a microservices into the Kubernetes cluster.
```
Comment (rhagarty) [inline README.md]: not quite sure what this means - can you re-phrase? At a minimum, second "microservices" should not be plural

File: README.md [hunk 2]
```diff
@@ -11,6 +11,15 @@ The [sample application](https://github.com/IBM/sample.microservices.web-app) us
...
+1. Deploy Prometheus server as a service into the Kubernetes cluster.
+1. Deploy Grafana as a service into the kubernetes cluster.
+1. Use ingress gateway to expose web application from the kubernetes cluster
```
Comment (rhagarty) [inline README.md]: Should we capitalize these names?

File: README.md [hunk 3]
```diff
@@ -152,16 +161,29 @@ Sample metrics graph for `thread count` on prometheus server:
 
 ### 5. Installing Grafana
 
-Grafana is a platform for analytics and monitoring. You can create different charts based on the metrics gathered by prometheus server. The deployment yaml file [prometheus server](manifests/deploy-grafana.yml) installs the grafana dashboard into the cluster which you can access on port 3000 after port forwarding.
+Grafana is a platform for analytics and monitoring. You can create different charts based on the metrics gathered by prometheus server. The deployment yaml file [prometheus server](manifests/deploy-grafana.yml) installs the grafana dashboard into the cluster which you can access on port 3000 after port forwarding. To run locally you can use the following command:
```
Comment (rhagarty) [inline README.md]: Capitalize names. Also, "gathered by the Prometheus server"

**Review by rhagarty — CHANGES_REQUESTED** [review]
small nits

File: README.md [hunk 4]
```diff
@@ -152,16 +161,29 @@ Sample metrics graph for `thread count` on prometheus server:
...
+
+![](images/grafana1.png)
+
 * Add Prometheus server URL.
+![](images/add-datasource.png)
+
 * Load the JSON file `[grafana dashboard](data/metrics-grafana-dashboard.json)`
+![](images/load-json.png)
+
 * View the Dashboard load the charts. The charts are real time. So, as you go through the webapp clicking each link the charts on the grafana dashboard will so spikes on each charts.
```
Comment (ykoyfman) [inline README.md]: *show spikes *chart

Comment (rhagarty) [reply README.md]: also, add tags to images

File: README.md [hunk 5]
```diff
@@ -146,22 +155,35 @@ kubectl port-forward pod/<prometheus-server-pod-name>  9090:9090
...
 * Add Prometheus server URL.
-* Load the JSON file `[grafana dashboard](data/metrics-grafana-dashboard.json)`
-* View the Dashboard load the charts. The charts are real time. So, as you go through the webapp clicking each link the charts on the grafana dashboard will so spikes on each charts.
+![Add datasource](images/add-datasource.png)
+
+* Import the JSON file [grafana dashboard](data/metrics-grafana-dashboard.json) into Grafana. This json file is the representation of what charts to display in the Grafana dashboard.
+![Import dashboard JSON](images/load-json.png)
+
+* The charts should be now loaded in the dashboard. The charts are real time. So, as you go through the webapp clicking each link, the charts on the Grafana dashboard will show spikes on each charts.
+![Grafana Metrics](images/grafana-metrics.png)
```
Comment (rhagarty) [inline README.md:185]: "on each chart"

**Review by rhagarty — CHANGES_REQUESTED** [review]
on more nit

**Review by rhagarty — APPROVED** [review]

**Pull request merged**